sponsor.html

```diff
@@ -0,0 +1,109 @@
+                        <a href="nutrition.html" title="Voeding">Voeding</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="sponsor.html" title="Sponsoren">Sponsoren</a>
+                    </div>
+                    <div class="menu-item menu-button">
+                        <a href="contact.html" title="Contact">Contact</a>
+                    </div>
+                    <div class="menu-item search-box">
+
+                        <input type="text" name="search" style="height: 60%; margin: 0 5px;" placeholder="Zoeken...">
+                    </div>
+                    <div class="social-wrapper">
+                        <a class="social-icon" href="https://www.facebook.com/bewustkopen">
+                            <img src="images/facebook.png" alt="Facebook" width="25" height="25">
+                        </a>
+                        <a class="social-icon" href="https://www.twtter.com/bewustkopen">
+                            <img src="images/twitter.png" alt="Twitter" width="25" height="25">
+                        </a>
+                        <a class="social-icon" href="https://www.pinterest.com/bewustkopen">
+                            <img src="images/pinterest.png" alt="Pinterest" width="25" height="25">
+                        </a>
+                    </div>
+                </div>
+                <div class="menu-button-mobile"></div>
+            </nav>
+            <div id="title-bar">
+                <h1 class="title">Sponsoren</h1>
+                <div class="breadcrumb"><a href="index.html" title="home">Home</a>-&gt;<a href="sponsor.html" title="Sponsor">Sponsor</a></div>
+            </div>
+            <div id="content">
+                <p>
+                    Om een zo breed mogelijk publiek te bereiken is samenwerking van groot belang voor Bewust Kopen. Samenwerking kan op verschillende manieren. We onderscheiden dit in twee groepen.</p>
+                <br>
+                <h4>Strategische partners:</h4>
+                <p>Hieronder verstaan we ondersteuning waar geen of beperkte financiele vergoeding aan gekoppeld is.</p>
+                <h4>Sponsoren:</h4>
+                <p>Sponsoren bieden financiele steun aan het bedrijf om verder te kunnen ontwikkelen.</p>
+                
+                <p>Op dit moment is onze grootste partner de <a href="https://www.vegetariers.nl/" title=vegetariersbond>vegetariersbond</a>. Zij zijn zowel strategisch partner alswel sponsor.</p>
+                <br>
+                <h4>Zelf sponsor of strategisch partner worden?</h4>
+                <p>Wij zijn altijd op zoek naar nieuwe samenwerkingen. DUs mocht u interesse hebben dan horen we dat graag van u. U kunt contact met ons opnemen per <a href="mailto:info@bewustkopen.nl" title="E-mail ons">mail</a> of <a href="contact.html" title="contact">telefoon</a>.</p>
+                <br>
+            <footer>
```

Comment:
De footer wordt niet goed weergegeven. Misschien ergens een div niet goed genest of afgesloten?